app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -29,10 +29,8 @@ const DeleteAlert = ({
     <Alert
       isOpen
       isSubmitting={deleting}
-      message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
+      message="Are you sure you want to delete the note? This action cannot be undone."
```

Comment:
`delete this note`. Not `the note`. We are showing this alert for a particular note. Thus we should use `this`.

Reply:
Ok @yedhink , I will correct this mistake.